tests/cli/main.cpp

```diff
@@ -438,6 +438,130 @@ BOOST_FIXTURE_TEST_CASE( cli_vote_for_2_witnesses, cli_fixture )
    }
 }
 
+BOOST_FIXTURE_TEST_CASE( cli_get_signed_transaction_signers, cli_fixture )
+{
+   try
+   {
+      INVOKE(upgrade_nathan_account);
+
+      // register account and transfer funds
+      const auto test_bki = con.wallet_api_ptr->suggest_brain_key();
+      con.wallet_api_ptr->register_account(
+         "test", test_bki.pub_key, test_bki.pub_key, "nathan", "nathan", 0, true
+      );
+      con.wallet_api_ptr->transfer("nathan", "test", "1000", "BTS", "", true);
+
+      // import key and save wallet
+      BOOST_CHECK(con.wallet_api_ptr->import_key("test", test_bki.wif_priv_key));
+      con.wallet_api_ptr->save_wallet_file(con.wallet_filename);
+
+      // create transaction and check expected result
+      auto signed_trx = con.wallet_api_ptr->transfer("test", "nathan", "10", "BTS", "", true);
```

Comment:
Please replace "BTS" with "1.3.0", otherwise the test case will fail when running in `testnet` branch.

Reply:
Done.